docs/docs/hydroflow_plus/quickstart/distributed.mdx

```diff
@@ -45,7 +45,7 @@ cargo run --example first_ten_distributed
 [() (process 1)] 6
 [() (process 1)] 7
 [() (process 1)] 8
-[() (process 1)] 9     
+[() (process 1)] 9
 ```
 
 You'll notice that our logs are not particularly descriptive, just showing `()` as an identifier. Furthermore, our processes have the same Rust type, which could lead to accidentally mixing up streams across the machines (this will throw an exception, but it would be nice to have a compile error).
```

Comment:
Should this be in the docs or in an Issue?

Reply:
This is meant to be educational prose, explaining why it is worth the effort to add location tags.

Reply:
Having worked through the code, I understand better. However you should check in the version without tags (which produces this output) so that readers working through the doc aren't confused.

Reply:
The docs should clarify that the user should create the files for `first_ten_cluster.rs` in both directories, then describe how to update `lib.rs` to include the line `pub mod first_ten_cluster;`.

Reply:
Ah! The template already includes the final version of this file, just `cargo generate` won't emit that right now because the PR isn't merged in yet.

Reply:
I'm suggesting that the template include the version without tags, since that's what you start the docs with. Are we agreed on that?

Reply:
Ooh I see yeah we could include both versions perhaps? Will add in a followup since it's for a different example than this PR.